packages/graphql/src/translate/queryAST/ast/QueryAST.ts

```diff
@@ -38,8 +42,17 @@ export class QueryAST {
         //         target: new Cypher.NamedVariable("this"),
         //     })
         // );
-
-        const result = this.operation.transpile({ returnVariable: new Cypher.NamedVariable("this") });
+        const queryASTEnv = new QueryASTEnv(neo4jGraphQLContext);
```

Comment:
Are we sure we want the context in the ENV? Maybe we should add the graphqlContext directly in the context

Reply:
I added directly to the Env because it will be relevant across the whole tree. We can definitely add it to the Context but we will re-assign the reference every time we push a new Context. At least this was my reasoning on the Env rather than in the Context!